### PR TITLE
i18n(zh-cn): translate `\guides\cms\contentful.mdx`

### PR DESCRIPTION
**Pull request by zreren**
### What kind of changes does this PR include?
Translated content
### Description
translate /guides/cms/contentful.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/contentful.mdx [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+  - **env.d.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+### 安装依赖项
+
+要连接到你的 Contentful 空间，请使用下面的命令使用你首选的包管理器同时安装以下两个包：
+- [`contentful.js`](https://github.com/contentful/contentful.js)，JavaScript的官方 Contentful SDK
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/contentful.mdx:69]:
```suggestion
- [`contentful.js`](https://github.com/contentful/contentful.js)，官方 Contentful JavaScript SDK
```

File: src/content/docs/zh-cn/guides/cms/contentful.mdx [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+    : import.meta.env.CONTENTFUL_DELIVERY_TOKEN,
+  host: import.meta.env.DEV ? "preview.contentful.com" : "cdn.contentful.com",
+});
+
+```
+
+上面的代码片段创建了一个新的 Contentful 客户端，将`.env`文件中的凭证传递进去。
+
+:::caution
+在开发模式下，你的内容将从 **Contentful 预览API**获取。这意味着你将能够从 Contentful Web 应用程序中查看未发布的内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/contentful.mdx:108]:
```suggestion
在开发模式下，你的内容将从 **Contentful 预览 API**获取。这意味着你将能够从 Contentful Web 应用程序中查看未发布的内容。
```

File: src/content/docs/zh-cn/guides/cms/contentful.mdx [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+
+1. **一个 Contentful 空间** - 对于本教程，我们建议从一个空的空间开始。如果你已经有一个内容模型，请随意使用它，但你需要修改我们的代码片段以与你的内容模型匹配。
+2. **集成了 [Contentful SDK](https://github.com/contentful/contentful.js)** 的 Astro 项目 - 有关如何在 Astro 项目中设置 Contentful 的详细信息，请参阅 [与 Astro 集成](#与-astro-集成)。
+
+### 设置 Contentful 模型
+
+在你的 Contentful 空间中，在 **内容模型** 部分，创建一个新的内容模型，并设置以下字段和值：
+
+- **Name:** 博客文章
+- **API identifier: ** `blogPost`
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/contentful.mdx:179]:
```suggestion
- **API identifier: ** `blogPost`
```
这里也渲染错误了

File: src/content/docs/zh-cn/guides/cms/contentful.mdx [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+  </head>
+  <body>
+    <h1>{title}</h1>
+    <time>{date}</time>
+    <article set:html={content} />
+  </body>
+</html>
+```
+
+在浏览器中导航到 http://localhost:3000/，然后点击其中一篇文章，以确保你的动态路由正常工作！
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/contentful.mdx:424]:
```suggestion
在浏览器中导航到 http://localhost:3000/， 然后点击其中一篇文章，以确保你的动态路由正常工作！
```
这里链接不太对

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**